### PR TITLE
create `zh-CN` translation `Glossary/WebDAV`

### PR DESCRIPTION
**Pull request by yin1999**
Add `zh-CN` translation for `Glossary/WebDAV`.

### CODE REVIEW

File: files/zh-cn/glossary/webdav/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ <li>复制和移动网页</li>
+ <li>锁定文档以免被多人编辑</li>
+</ul>
+
+<dl>
+</dl>
+
+<h2 id="了解更多">了解更多</h2>
+
+<h3 id="基础知识">基础知识</h3>
```
Comment (t7yang) [inline files/zh-cn/glossary/webdav/index.html:28]: remove this block of content.

File: files/zh-cn/glossary/webdav/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ <li>维基百科上的 {{interwiki("wikipedia", "WebDAV")}}</li>
+</ul>
+
+<h3 id="规范">规范</h3>
+
+<ul>
+ <li><span class="external">{{rfc(2518)}}</span></li>
+ <li><span class="external">{{rfc(3253)}}</span></li>
+ <li><span class="external">{{rfc(3744)}}</span></li>
+</ul>
```
Comment (t7yang) [inline files/zh-cn/glossary/webdav/index.html:40]: update this block of content according this [section](https://github.com/mdn/content/blob/4a9f9d2e9c1811ce3519a7061e906db9200a5e97/files/en-us/glossary/webdav/index.md#see-also) in `HTML` form.

**Review by t7yang — CHANGES_REQUESTED** [review]

File: files/zh-cn/glossary/webdav/index.html (hunk 2 above)
Comment (yin1999) [reply files/zh-cn/glossary/webdav/index.html:40]: any example?

Comment (irvin) [reply files/zh-cn/glossary/webdav/index.html:40]: 請參考目前的[翻譯測試](https://pr3046.content.dev.mdn.mozit.cloud/zh-CN/docs/Glossary/WebDAV)頁與[英文版網頁](https://developer.mozilla.org/en-US/docs/Glossary/WebDAV)，你直接調整讓其 `<ul><li>` 階層格式一致即可

**Review by t7yang — APPROVED** [review]

**Pull request merged**